version_check.sh

```diff
@@ -22,10 +22,10 @@ elif [ "$1" = "ps80" ]; then
   version=${PS80_VER}
   release=${PS80_VER#*-}
   revision=${PS80_REV}
-elif [ "$1" = "ps81" ]; then
-  version=${PS81_VER}
-  release=${PS81_VER#*-}
-  revision=${PS81_REV}  
+elif [ "$1" = "ps81" -o "$1" = "ps82" ]; then
```

Comment:
The same as above - we will have to update the scripts durung every release and we want to overcome this,